torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1116,6 +1178,22 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
            sample_inputs_func=sample_inputs_fliplr_flipud,
            test_inplace_grad=False,
            supports_tensor_out=False),
+    OpInfo('linalg.norm',
+           op=torch.linalg.norm,
+           dtypes=floating_and_complex_types_and(torch.float16, torch.bfloat16),
+           test_inplace_grad=False,
+           test_complex_grad=True,
+           supports_tensor_out=True,
+           decorators=[skipCUDAIfNoMagma, skipCPUIfNoLapack],
+           sample_inputs_func=sample_inputs_linalg_norm,
+           aten_name='linalg_norm',
+           skips=(
+               # TODO: remove this once `pow` is implemented for float16
```

Comment:
it would be awesome if you could file an issue and link it here

Reply:
#50789